testing/testing.h

```diff
@@ -1,5 +1,5 @@
 #pragma once
-#include <string>
+#include <sstream>
```

Comment:
What is the reason for making this a string stream?
I tend to not like string streams very much.
In an ideal world i'd love to be able to just set an error message with printf style.
What do you think?
Might it be work making a specific error object, or some other approach?